modules/installation-osp-deploying-bare-metal-machines.adoc

```diff
@@ -18,10 +18,14 @@ file. Your cluster can have both control plane and compute machines running on b
 
 Bare-metal compute machines are not supported on clusters that use Kuryr.
 
+[WARNING]
+====
+On {rh-openstack} 16.2, clusters that use bare metal workers and the OpenShift SDN can fail depending on the MTU configuration that is described in a link:https://bugzilla.redhat.com/show_bug.cgi?id=2153801[known issue].

```

Comment:
This issue is on_qa and it's likely to be verified soon.

Reply:
How soon? Might it make sense to put a hold on this change?

Reply:
Sorry for the delay. Given the bug was already verified I will remove it from this docs PR
https://bugzilla.redhat.com/show_bug.cgi?id=2177155